hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/OzoneRepair.java

```diff
@@ -31,7 +31,8 @@
  * Ozone Repair Command line tool.
  */
 @CommandLine.Command(name = "ozone repair",
-    description = "Operational tool to repair Ozone",
+    description = "Repair is an advanced operation and the nodes being repaired " +
+        "should be stopped while it is running.",
```

Comment:
As this is a command description, it would be better to change how we are mentioning the condition of nodes being stopped:
```suggestion
    description = "Advanced tool to repair ozone for which the nodes being repaired " +
        "should be stopped while it is running.",
```

Reply:
```suggestion
    description = "Advanced tool to repair Ozone. The nodes being repaired " +
	        "must be stopped before the tool is run.",
```